src/actions/index.js

```diff
@@ -117,13 +119,21 @@ export function signOut() {
   }
 }
 
-export function fetchProjects(parms) {
+export function fetchProjects() {
   return dispatch => {
     dispatch(setError(''))
-    dispatch(setIsFetching(false))
+    var callFetchProjects = tag => dispatch(fetchProjectsByTag(tag.value))
+    forEach(callFetchProjects, filter(propEq('display', true), GLOBALS.DISCIPLINES))
+  }
+}
+
+
+export function fetchProjectsByTag(tag) {
+  const parms = {id: MOBILE_PROJECTS, cards: true, tags: tag, sort: 'display_name'}
```

Comment:
Right. I saw this was pointing to `mobile_projects` file, I was just curious what those project IDs represent when it isn't set to production in that ternary. I guess that's something to think about but in no way blocking for this PR.